### ISSUE TITLE
model is overfitted?

### ISSUE THREAD

**Issue by RamatovInomjon** [3d-hand-shape/hand-graph-cnn#23]
Hi @3d-hand-shape , 
First of all , thanks for your great work!
However, I've got a problem with using your model, i managed to change the code to test my own data, I tried with hand detector and without hand detector , result is really bad. here I am sharing two output, the result is in your data is good, but in other images is bad
Please help,

**Comment by RamatovInomjon** [comment]
![test1](https://user-images.githubusercontent.com/43458270/68357578-c86fc100-0150-11ea-868d-d640df4bdcca.jpg)
![test2](https://user-images.githubusercontent.com/43458270/68357587-cd347500-0150-11ea-81eb-105f6cb5f583.jpg)

**Comment by RamatovInomjon** [comment]
Hi, what padding size did you choose for bbox?

**Comment by VinhDuong10998** [comment]
Hi @RamatovInomjon ,
How do you calc pose_root and bbox when you run your image?
Can share with me?

**Comment by Arrowa70** [comment]
Hi @RamatovInomjon , I try to modify the code for my own testing too, but not done yet. I see your result and I wonder if it really can't work well in actual application.  Maybe just because some conditions are not taken into account. Could you please  share your code ? I'd be really grateful.

**Comment by 3d-hand-shape** [comment]
Hi @RamatovInomjon , thanks for sharing your results. This release is just to reproduce the results in the paper. I suggest you finetune the model on your own data to get better results.
